src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -562,8 +562,6 @@ def test_assert(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
-    environment.remove_key('ASSERTS_HAVE_SECURITY_IMPLICATION')
```

Comment:
can just remove all of these environment.remove_key from this file ?

Reply:
Also, where did the test go ? is it intentional to remove it ? can keep it with chrome frame, does not hurt.

Reply:
I removed it intentionally. We already know that IGNORE_CONTAINS works, so seems redundant to include another test for it.

Reply:
And yeah, all of those should be removed. I thought I had checked but I'll do another pass and clean those up.